charts/piraeus/Chart.yaml

```diff
@@ -9,3 +9,7 @@ dependencies:
   - name: "etcd"
     version: "0.7.5"
     condition: etcd.enabled
+  - name: "piraeus-ha-controller"
+    version: "1.1.4"
+    repository: "https://piraeus.io/helm-charts/"
+    condition: ha-controller.enabled
```

Comment:
Should be `piraeus-ha-controller.enabled`.

Reply:
corrected.